### ISSUE TITLE
Bug: Invisible close paren from stylish formatter

### ISSUE THREAD

**Issue by bertie-wheen** [eslint/eslint#16390]
### Environment

Node version: 16.13.2
npm version: 8.1.2
Local ESLint version: 8.24.0
Global ESLint version: -
Operating System: Windows 11 (64 bit)

### What parser are you using?

vue-eslint-parser

### What did you do?

<details>
<summary>Configuration</summary>

This is probably nothing to do with Vue, but:
1. Scaffold an app using `npm init vue@latest`. (This is probably nothing to do with Vue, but that's the context I encountered it in.)
2. Select only ESLint from the options. (Or rather, at least ESLint: I doubt selecting additional options would make any difference.)
3. Enter the app directory, and `npm i`.
4. Add `rules: { 'max-len': ['warn', 120] }` to `.eslintrc.js`. (This is pretty arbitrary, it's just to generate some problems to make the message appear.)
</details>

```js
> npm run lint
```

### What did you expect to happen?

```
> <app>@0.0.0 lint
> eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore


<dir>\<app>\src\components\icons\IconCommunity.vue
  4:1  warning  This line has a length of 918. Maximum allowed is 120  max-len

<dir>\<app>\src\components\icons\IconDocumentation.vue
  4:1  warning  This line has a length of 1118. Maximum allowed is 120  max-len

<dir>\<app>\src\components\icons\IconEcosystem.vue
  4:1  warning  This line has a length of 1841. Maximum allowed is 120  max-len

<dir>\<app>\src\components\icons\IconSupport.vue
  4:1  warning  This line has a length of 152. Maximum allowed is 120  max-len

<dir>\<app>\src\components\icons\IconTooling.vue
   1:1  warning  This line has a length of 156. Maximum allowed is 120  max-len
  15:1  warning  This line has a length of 411. Maximum allowed is 120  max-len

✖ 6 problems (0 errors, 6 warnings)
```

### What actually happened?

I copy-pasted the output from Powershell into the box above, and you'll notice that there's a closing parenthesis after the `6 warnings`... except it doesn't appear in the shell itself (except it must be there, albeit invisible, to have appeared in the copy-pasted output). In my actual terminal (which uses Unifont), the fancy cross symbol is two characters wide, and somehow this (at least I assume it's this that) makes the close paren disappear, which means that what I see is basically:
```
✖ 6 problems (0 errors, 6 warnings
```

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

The message is from [the stylish formatter](https://github.com/eslint/eslint/blob/main/lib/cli-engine/formatters/stylish.js#L87) (I think).

PS Some extra weirdness in this issue form itself! When I'm editing a text box with the cross symbol inside it, it looks like a normal diagonal cross (and is, incidentally, about two-and-a-half characters wide...), but then when I focus a different textarea, it becomes a plus sign, despite the character code being for a 'Heavy Multiplication X Emoji'... and weirder still, I've just noticed that it's a plus sign in the title of this Chrome tab: [https://unicode-table.com/en/2716/]()

Maybe a different Unicode cross might be better?
